app/controllers/login_controller.rb

```diff
@@ -5,28 +5,26 @@ def new
     def create
         user =  Admin.find_by(name: login_params[:name])
 
-        if user && user.authenticate(login_params[:password])
+        respond_to do |format|
+            if user && user.authenticate(login_params[:password])
             
-            token = SecureRandom.hex()
-            session = Session.create(user_id: user.id, token:token)
-            cookies.signed[:session_token] = { value: token, httponly: true, secure: Rails.env.production? }
-            redirect_to pages_path, notice: "Login Successfully"
-            # render json: {token:token, user_id: session[:user_id]}
-        else 
-            redirect_to root_path, notice: "Invalid username or password"
-            # render :new, notice: "login failed"
-        end 
+                session[:user_id] = user.id
```

Comment:
wrong indentation  It should be only 2 spaces

Reply:
indentation is corrected